modules/ROOT/pages/provisioning-nutanix.adoc

```diff
@@ -59,4 +59,96 @@ EOF
 
 == Launching a VM instance
 
-You can provision an FCOS instance using the Nutanix Prism Central web portal or via the Prism Central API with `curl`. Ignition configuration can be passed to the VM as a "cloud-init custom script".
+You can provision an FCOS instance using the Nutanix Prism Central web portal or via the Prism Central API with `curl`. Ignition configuration can be passed to the VM as a "cloud-init custom script". You can use the Nutanix Prism Central API to launch a VM in Nutanix:
```

Comment:
The last sentence repeats the first one.  Maybe something like `For example, to launch a VM using the API:`?